editor/src/components/canvas/canvas-strategies/canvas-strategies.spec.browser2.tsx

```diff
@@ -274,7 +274,7 @@ describe('Strategy Fitness', () => {
           data-uid='bbb'
         />
         <div
-          style={{ backgroundColor: '#0091FFAA', width: 100, height: 50 }}
+          style={{ backgroundColor: '#0091FFAA', width: 100, height: 50, contains: 'layout' }}
```

Comment:
This should be `contain`, not `contains`

Reply:
so how does it pass lol

Reply:
You tell me! https://developer.mozilla.org/en-US/docs/Web/CSS/contain

Reply:
fixed